tutorials/hist/h1readanddraw.C

```diff
@@ -33,6 +34,7 @@ void h1draw()
    if (!example) return;
 
    example->ls();
```

Comment:
All lines above can be replaced with just reading `hsimple.root` which is generated during ROOT's build.

`example->ls()` does not add anything, or if we want to keep it, must be accompanied with a comment: `// Sohw the file contents` or similar.

Reply:
Me and Olivier talked about this and he said that the lines above ensure that a hsimple.root file exists. Incase someone is using like a clone of root or something.

Reply:
@couet hsimple.root is generated during ROOT's build, it's always in `TutorialDir`, we don't need to run the script.

And I'd say `ls()` needs a comment or needs to be removed: it's not part of reading and drawing a histogram.